creator-node/nginx_conf/nginx.conf

```diff
@@ -29,6 +29,18 @@ http {
 					max_size=4g inactive=12h use_temp_path=off;
     proxy_read_timeout 3600; # 1 hour in seconds
 
+    upstream exporter-postgres {
```

Comment:
where's code to scrape these exporters in prometheus?

Reply:
That work will be handled under INF-280.

Reply:
cool